core/domain/suggestion_registry.py

```diff
@@ -305,9 +305,9 @@ def populate_old_value_of_change(self) -> None:
             'Subclasses of BaseSuggestion should implement '
             'populate_old_value_of_change.')
 
-    # Here we use type Any because sub-classes of BaseSuggestion can have
-    # different Change classes. So, to allow every Change class we used Any
-    # type here.
+    # TODO(#16047): Here we use type Any because sub-classes of BaseSuggestion
```

Comment:
TODOs generally need to be an instruction or have an instruction within them. This one doesn't seem to have one.

Reply:
Comment updated but not able fit all info because they are lots of info. PTAL!